KeyPoint2/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
Update gitignore with target, .classpath, .settings, .project and all the keypoints related with previous topics